WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3564,27 +3566,25 @@ e_byzantium = {
 		
 		capital = 560 # Cherson
 		
-		hellenic_pagan = 600
-		bosporan_faith = 200
+		hellenic_pagan = 200
+		bosporan_faith = 600
```

Comment:
I'm not opposed to raising the Hypsistarian faith, but I disagree on your other change. Tauricia was an important part of the Hellenic world and of religious importance, justifying this value which comes from ACR.